### PR TITLE
[Duplicate of #7016] - Creating User Authentication UI with Compose for Desktop

### LINKED ISSUE #6706: Creating User Authentication UI(Login and Signup) with Compose Desktop
### Topic Suggestion
[Kotlin] Creating User Authentication UI(Login and Signup) with Compose Desktop

#### Proposed title of article
Creating User Authentication UI(Login and Signup) with Compose Desktop

#### Proposed article introduction
With Compose Multiplatform, one can create a very interesting layout using Kotlin and Jetpack compose for Desktop applications. Having a good user authentication layout attracts users to sign in to your system easily. Compose desktop has made this achievable with a lot of ease unlike SWINGS in Java which requires one to write lots of code. This tutorial will help the reader to understand how to use Compose Desktop to create well-structured desktop applications just like in Android. 

#### Key takeaways 
- How to use Compose Desktop.
- How to implement Layout designs using Compose Desktop.
- Compose desktop design patterns

#### Article quality
This article is going to be unique since Compose Desktop is a new technology that is just coming into existence. it is going to help the learner to get started with Compose Desktop. It will help the learner to also migrate from using the tiresome SWING library in Java. 

#### References
[Compose Desktop](https://github.com/JetBrains/compose-jb/tree/master/tutorials/Getting_Started)

### PR DESCRIPTION
**Pull request by Collince-Okeyo**
#### Formatting and structure checks
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**

- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Is the article you are submitting an in-depth and unique article? Does it go beyond what is in the official docs and what is covered in other blog sites. See these articles as [examples](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md).
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**

- [x] Have you included a hero image that is **600x400 pixels**, under **300KB in size**, and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
      
#### Grammar & plagiarism checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) - [Grammarly](https://grammarly.com) 
- [x] Please place your article through a 3rd party plagiarism checker. We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**
- [x] Have you checked your code runs correctly and you've highlighted all necessary dependencies for installation?
- [x] Are the software programs and packages you're highlighting in your article up to date, using current versions and not deprecated?

This closes #6706

### CODE REVIEW

**Comment by Collince-Okeyo** [comment]
Hello @mohamedgh16, am sorry the PR was closed due to some grammar checks that I failed to correct. I have reopened it please continue with the review.

**Comment by lalith1403** [comment]
plagiarism check done ✔️

**Comment by mohamedgh16** [comment]
@Collince-Okeyo please consider the corrections to minimize revision next time.

File: content/articles/compose-for-desktop/index.md [hunk 1]
```diff
@@ -0,0 +1,352 @@
...
+```
+
+Since Kotlin is considered a multi-platform language, you can build a whole desktop application with Kotlin only. In this tutorial, we have learned how to build user authentication screens. 
+
+This project can be found in my [GitHub](https://github.com/Collince-Okeyo/UserAuthentication) repositories. You can fork and clone it in your machine and learn how to create the screens with Compose for Desktop implementation. 
+
+To learn more about building desktop apps with Compose for Desktop visit [here](https://github.com/JetBrains/compose-jb/tree/master/tutorials/Getting_Started).
+
+### Conclusion
+To learn more about building desktop apps with this technology, you will have to do a lot of practice. If you love creating web applications, with Compose for Web, you can create awesome websites as well.
```
Comment (mohamedgh16) [inline content/articles/compose-for-desktop/index.md:352]: Add a further reading section.

Comment (Collince-Okeyo) [reply content/articles/compose-for-desktop/index.md]: > Add a further reading section.

@mohamedgh16 , I have added further reading as suggested.

**Comment by hectorkambow** [comment]
@Collince-Okeyo any update on this PR - are any more edits being made? 


cc @mohamedgh16

**Comment by Collince-Okeyo** [comment]
> @Collince-Okeyo any update on this PR - are any more edits being made?
> 
> cc @mohamedgh16

@hectorkambow , I had completed making the suggested edits.

**Comment by hectorkambow** [comment]
Will be jumping on this one. 

@mohamedgh16 NOTE: any PR that is unattended this long - please be sure to bring to our attention ASAP - ping us for us to get another notification. 

It is the Peer Reviewers responsibility to make sure their work get pushed through so they can pick up another review.

**Comment by hectorkambow** [comment]
@Collince-Okeyo & @mohamedgh16 

This article was pretty far from done (or ready to be published).

**Comment by hectorkambow** [comment]
👀

**Comment by Collince-Okeyo** [comment]
> @Collince-Okeyo & @mohamedgh16
> 
> This article was pretty far from done (or ready to be published).

@hectorkambow , let me look at it once again.

**Pull request closed without merging**